roles/ovirt-host-deploy-firewalld/README.md

```diff
@@ -0,0 +1,46 @@
+oVirt host deploy - firewalld
+=============================
+
+This role is used for configuration of firewalld on oVirt hypervisor.
+
+Requirements
+------------
+
+ * Ansible version 2.0
+
+Role Variables
+--------------
+
+| Name                        | Default value  | Description                                                       |
+|-----------------------------|----------------|-------------------------------------------------------------------|
+| host_deploy_cluster_version | 4.2            | The version of the cluster where the host resides. Based on this version the role will enable cluster version specific firewalld rules. |
+| host_deploy_gluster_enabled | false          | If true special virt firewalld rules will be enabled on host.     |
+| host_deploy_virt_enabled    | false          | If true special gluster firewalld rules will be enabled on host.  |
+| host_deploy_vdsm_port       | 54321          | VDSM port. This port will be enabled by firewalld.                |
```

Comment:
no, it's `ansible_ssh_port` variable, which is used by ansible by default. It's not this role specific, but ansible specific.